.circleci/generate_circleci_config.py

```diff
@@ -55,6 +55,7 @@ def main():
     template_env = Environment(loader=FileSystemLoader(searchpath=circle_directory), autoescape=True)
     template_env.filters['dev_releases'] = dev_releases
     template_env.filters['get_airflow_version'] = get_airflow_version
+    template_env.filters['checkair2'] = checkair2
```

Comment:
```suggestion
    template_env.filters['is_airflow_2'] = is_airflow_2
```

and use `is_airflow_2` in the config file as it better in terms of naming